ui/pages/permissions-connect/snaps/snaps-connect/snaps-connect.js

```diff
@@ -57,7 +58,11 @@ export default function SnapsConnect({
   const snaps = getDedupedSnaps(request, currentPermissions);
 
   const SnapsConnectContent = () => {
-    const { hostname: trimmedOrigin } = useOriginMetadata(origin) || {};
+    let trimmedOrigin = (useOriginMetadata(origin) || {})?.name;
```

Comment:
```suggestion
    let trimmedOrigin = (useOriginMetadata(origin) || {})?.hostname;
```

We used hostname before, should we not keep that?

Reply:
Good catch, that was unintended. Fixed here: [19e8182](https://github.com/MetaMask/metamask-extension/pull/26715/commits/19e8182ef6a0e7f597547c40799d825b47ab3de1)